test/Directory.build.props

```diff
@@ -0,0 +1,42 @@
+<Project>
+  <PropertyGroup>
+    <DisableImplicitFrameworkReferences Condition=" '$(TargetFramework)' == 'net452' ">true</DisableImplicitFrameworkReferences>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">

```

Comment:
Is it weird that these `TargetFramework` conditions are in here when the actual target frameworks themselves are defined in the project file?